src/routes/buildManifestObject.js

```diff
@@ -104,15 +105,17 @@ export function buildManifestObject() {
       type: 'okapi',
       path: 'inventory/instances',
       records: 'instances',
-      accumulate: true,
+      accumulate: false,
       fetch: false,
+      resultDensity: 'sparse',
     },
     holdings: {
       type: 'okapi',
       path: 'holdings-storage/holdings',
       records: 'holdingsRecords',
-      accumulate: true,
+      accumulate: false,
```

Comment:
@JohnC-80 is there a reason you changed it for holdings?

Reply:
Sorry, wasn't sure where I needed to change it, exactly... will put it back!